### DISCUSSION TITLE
Optimizing History Command Display in WSL

### DISCUSSION THREAD

**Discussion by hurricanedjp** [marlonrichert/zsh-autocomplete#644] [Q&A]
Thank you for the outstanding project. 
I have a minor issue while using it. I followed the instructions in the [readme](https://github.com/marlonrichert/zsh-autocomplete#limit-the-number-of-lines-shown) and set 'list-lines' for the History menu to 256, but it displays too much on the entire screen. I want to be able to find 256 history commands without occupying the entire screen. Instead, I would like to set it to display 8 lines and navigate through them using paging. How can I achieve this?

 PS: I am using the WSL (Windows Subsystem for Linux) system.

![image](https://github.com/marlonrichert/zsh-autocomplete/assets/19743827/494e21dc-979a-4a0c-9dbe-d1e67f71aa7e)

**Comment by marlonrichert** [comment]
> I would like to set it to display 8 lines and navigate through them using paging. How can I achieve this?

Sorry, not possible. It's a limitation in Zsh. You could submit a feature request for being able to limit the max number of rows the completion menu occupies on screen, even when there's more completions than that, to the [official Zsh mailing list](https://zsh.sourceforge.io/Arc/mlist.html).
